### DISCUSSION TITLE
Support for Intellij 2022.2 EAP family

### DISCUSSION THREAD

**Discussion by azhe403** [community/community#17045] [Copilot Conversations] [answered]
Jetbrains already opening Intellij 2022.2 EAP. 
As a Beta lover and Copilot lover, we need Copilot support for this version of Intellij platform.

https://blog.jetbrains.com/idea/2022/05/intellij-idea-2022-2-eap-1/
https://plugins.jetbrains.com/plugin/17718-github-copilot/versions

thanks ❤️❤️

**Comment by jansorg** [comment, marked as answer]
It's under way and should be available later this week.
